repo/packages/C/cassandra/0/marathon.json

```diff
@@ -0,0 +1,47 @@
+
+{
+  "id": "/cassandra/{{framework/name}}",
+  "instances": 1,
+  "cpus": 0.5,
+  "mem": 512,
+  "ports": [
+    0
+  ],
+  "uris": [
+    "https://downloads.mesosphere.io/cassandra-mesos/artifacts/0.1.0-SNAPSHOT-390-rewrite-825756a084/cassandra-mesos-0.1.0-SNAPSHOT-390-rewrite-825756a084.tar.gz",
+    "https://downloads.mesosphere.io/java/jre-7u76-linux-x64.tar.gz"
+  ],
+  "labels": {
```

Comment:
Please remove all of these labels.  When using the DCOS CLI, these and more will be added automatically.  Other clients probably don't want them either.